source/modules/configuration/crontasks.rst

```diff
@@ -33,7 +37,15 @@ To execute a specific action from the command-line, you can run (for mailgate ac
 
    php GLPI/front/cron.php --force mailgate
 
+
+For Linux/MacOS, you should should use the web server's user's (**www-data, apache, etc**):
+
+::
+
+   sudo -u www-data php GLPI/front/cron.php --force mailgate
+
 You will need to replace `GLPI` with the path to your GLPI folder.
+You may also need to replace `php` with the full path to your php binary if it is not in the PATH (`/usr/bin/php`).
```

Comment:
Should suggest replacing it with the path to the specific version of PHP loaded in the web server for GLPI, whether that be the Apache module, FPM, etc.